oauth/authcode.go

```diff
@@ -128,12 +128,13 @@ func (ac *AuthorizationCodeTokenSource) Token() (*oauth2.Token, error) {
 // AuthCodeHandler sets up the OAuth 2.0 authorization code with PKCE authentication
 // flow.
 type AuthCodeHandler struct {
-	ClientID     string
-	AuthorizeURL string
-	TokenURL     string
-	Keys         []string
-	Params       []string
-	Scopes       []string
+	ClientID      string
+	AuthorizeURL  string
+	TokenURL      string
+	Keys          []string
+	Params        []string
+	Scopes        []string
+	DefaultValues map[string]string
```

Comment:
Does this need to get initialized somewhere? If we rely on it to happen in someone's `main` then what happens if they forget?